lib/workers/repository/update/branch/index.ts

```diff
@@ -570,9 +570,23 @@ export async function processBranch(
       await scm.checkoutBranch(config.baseBranch);
       updatesVerified = true;
     }
-    // istanbul ignore if
-    if (branchPr && platform.refreshPr) {
-      await platform.refreshPr(branchPr.number);
+
+    if (branchPr) {
+      const platformOptions = getPlatformPrOptions(config);
+      if (
+        platformOptions?.usePlatformAutomerge &&
```

Comment:
It's mystifying at the moment me how `platformOptions` could be undefined right here, but `lib/workers/repository/update/branch/index.spec.ts` fails if I don't use optional chaining in this spot...

Reply:
Removing the upper check on `config.autoMergeAllowed` doesn't fix coverage. Going into a block of meetings, but will come back to this later

Reply:
Experimentally verified `platformOptions === undefined` here and `getPlatformPrOptions()` doesn't seem to be called (a throw inside it doesn't change the outcome of `lib/workers/repository/update/branch/index.spec.ts` tests). I don't use Typescript at my day job, so maybe I'm making some rookie mistake?

Reply:
Ahhh, Jest mocks! Another thing I don't use at my day job 😅 So when `lib/workers/repository/update/branch/index.spec.ts` runs, the `lib/workers/repository/update/pr` module is mocked away, including `getPlatformPrOptions`.

This is telling me I do have a very real gap in my test coverage here...